spec/index.adoc

```diff
@@ -962,6 +964,14 @@ For periodic or time-based execution, the WebAssembly module can export a functi
 
 NOTE: While an implementation will likely try to keep the interval between `canister_heartbeat` invocations to within a few seconds, this is not formally part of this specification.
 
+==== Global timer
+
+For time-based execution, the WebAssembly module can export a function with name `canister_global_timer`. This function is called if the canister has set its global timer (using the System API function `ic0.global_timer_set`) and the current time (as returned by the System API function `ic0.time`) has passed the value of the global timer. Once the function `canister_global_timer` is scheduled, the canister's global timer is reset. The global timer is also reset upon changes to the canister's Wasm module (calling `install_code`, `uninstall_code` methods of the management canister or if the canister runs out of cycles). In particular, the function `canister_global_timer` won't be scheduled again unless the canister sets the global timer again (using the System API function `ic0.global_timer_set`). The global timer scheduling algorithm is implementation-defined.
+
+`canister_global_timer` is triggered by the IC, and therefore has no arguments, no caller, and cannot reply or reject.
```

Comment:
Maybe it's worth explicitly mentioning that we can still initiate new calls?

Reply:
> we can still initiate new calls

You mean it's possible to initiate new calls from the `canister_global_time` function? Then this remark would apply analogously to `canister_heartbeat`.

Reply:
Yes, we can initiate new calls both from the heartbeat and the timer. Up to you though, as technically we are correct.